util/logtools/encounter_tools.ts

```diff
@@ -358,7 +362,7 @@ class TLFuncs {
     return str.split(' ').map((str) => {
       const cap = str[0]?.toUpperCase();
       const lower = str.slice(1);
-      if (cap)
+      if (cap !== undefined && cap.length > 0)
```

Comment:
```suggestion
      if (cap !== undefined)
```

Unnecessary, I think?

Reply:
Realistically, this whole function should *probably* be switched over to use Valarnin's implementation that's living in `resources/string_handlers` now. I'm not sure how I missed doing that during the move.